helper/iplookuptable.c

```diff
@@ -83,9 +83,9 @@ typedef struct trie_node {
 } trie_node_t;
 
 /** Number of L2\L3 entries(subtrees) per cache cube. */
-#define CACHE_NUM_SUBTREE	(1 << 13)
+#define CACHE_NUM_SUBTREE	(4 * 1024)
```

Comment:
This code created pools with 1M events without checking capability. My other patches decrease pool capability to less than 1M events, so pool create started to fail. Also the code assumed that a queue can hold all 1M events. The default queue size is actually 4k events, so this code has a long time created 1M pool and successfully used only first 4k events of it (those that fit into a queue).

I decreased required sizes to 4k (which seems to have worked so far) and added capability checks, so that 'make check' passes and proper error is reported is capability is less than 4k.